ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/switch/v1/switch/clientlibs/editor/js/editDialog.js

```diff
@@ -0,0 +1,52 @@
+/*******************************************************************************
+ * Copyright 2023 Adobe
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+(function($) {
+  "use strict";
+
+  var EDIT_DIALOG = ".cmp-adaptiveform-switch__editdialog",
+      ENABLE_UNCHECKED_VALUE = EDIT_DIALOG + " .cmp-adaptiveform-switch__enable-unchecked-value",
+      ENUM_OPTION = ".cmp-adaptiveform-switch__enums coral-multifield-item",
+      Utils = window.CQ.FormsCoreComponents.Utils.v1;
+
+  function _handleLayout(dialog) {
```

Comment:
This looks completely static and violates the structure (HTML) vs presentation (CSS) desig,. Ideal approach should be to separate the structure and presentation, making code cleaner and more maintainable.